app/create_buyer/forms/auth_forms.py

```diff
@@ -8,7 +8,7 @@ class EmailAddressForm(FlaskForm):
     email_address = DMStripWhitespaceStringField(
         "Your email address",
         validators=[
-            DataRequired(message="You must provide an email address"),
-            ValidEmailAddress(message="You must provide a valid email address"),
+            DataRequired(message="Enter an email address"),
+            ValidEmailAddress(message="Enter an email address in the correct format, like name@example.com"),
```

Comment:
I wonder if the example we provide should look like its from a public sector organisation, something like `name@gov.example` or `name@example.gov.uk` perhaps. Might be overthinking it though...

Reply:
I agree - buyers are unlikely to have `.com` emails

Reply:
Done!